.github/workflows/swiftlint.yml

```
@@ -0,0 +1,35 @@
+      - '**/*.swift'
+      - '.swiftlint.yml'
+  push:
+    branches: [ main ]
+    paths:
+      - '**/*.swift'
+      - '.swiftlint.yml'
+
+jobs:
+  swiftlint:
+    runs-on: macos-latest
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)